crypto/evp/encode.c

```diff
@@ -453,12 +458,43 @@ static int evp_decodeblock_int(EVP_ENCODE_CTX *ctx, unsigned char *t,
         *(t++) = (unsigned char)(l) & 0xff;
         ret += 3;
     }
+
+    /* process the last block that may have padding. */
+    a = conv_ascii2bin(*(f++), table);
+    b = conv_ascii2bin(*(f++), table);
+    c = conv_ascii2bin(*(f++), table);
+    d = conv_ascii2bin(*(f++), table);
```

Comment:
This reads out of bounds is `n == 0` on input (or after stripping trailing whitespace, ...).

Reply:
True, thanks. I added the check.